xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -479,7 +479,7 @@ bool CGUIWindowVideoBase::ShowInfo(const CFileItemPtr& item2, const ScraperPtr&
   do
   {
     // reload images
-    //! !todo we need this to update images in video info dialog immediatly after refresh, but why?
+    //! !todo we need this to update images in video info dialog immediately after refresh, but why?
```

Comment:
while you are at it, could you change `!todo` to `@todo`?

Reply:
done